src/Components/Nav/Nav.js

```diff
@@ -45,14 +46,12 @@ export default class Nav extends Component {
                 <NavbarToggler onClick={this.toggleCollapsed} />
                 <Collapse isOpen={this.state.collapsed} navbar>
                     <BootstrapNav pills={true}>
-                        {this.state.items.map(item => {
-                            return (
-                                <NavItem key={item.folder}>
-                                    <NavLink tag={Link} active={item.href === window.location.pathname} to={item.href}>
-                                        {item.title}
-                                    </NavLink>
-                                </NavItem>
-                            );
+                        {this.state.items.map((item, index) => {
+                            if (item.children.length && item.type === 'folder'){
+                                return <NavDropDown key={index} options={item}/>
```

Comment:
Each item in the response have a `hash` property that is different for each (like an id) when you have that kind of property in what you render you use those as a key, the index is "ok" tho.